pyproject.toml

```diff
@@ -46,19 +50,32 @@ classifiers = [
   "Topic :: Scientific/Engineering",
   "Topic :: Software Development",
 ]
+# core dependencies of sktime
+# this set should be kept minimal!
 dependencies = [
-  "numpy<1.27,>=1.21",
-  "packaging",
-  "pandas<2.2.0,>=1.1",
-  "scikit-base<0.7.0",
-  "scikit-learn<1.4.0,>=0.24",
-  "scipy<2.0.0,>=1.2",
+  "numpy<1.27,>=1.21",  # required for framework layer and base class logic
+  "packaging",  # for estimator specific dependency parsing
```

Comment:
Does not correspond to PR title, but may be worth to add lower and upper bounds based on previous discussions?

Reply:
what do you mean? All I added were the comments. If the deps change, that's a mistake on my side, but I can see no changes?

Reply:
That's what I meant by "Does not correspond to PR title". Resolving both comments.